rotkehlchen/exchanges/iconomi.py

```diff
@@ -258,10 +259,39 @@ def query_balances(self, **kwargs: Any) -> ExchangeQueryBalances:
 
         for balance_info in resp_info['daaList']:
             ticker = balance_info['ticker']
-            self.msg_aggregator.add_warning(
-                f'Found unsupported ICONOMI strategy {ticker}. '
-                f' Ignoring its balance query.',
-            )
+
+            if ticker == 'AUSTS':
+                # The AUSTS strategy is 'ICONOMI Earn'. We know that this strategy holds its
+                # value in Anchor UST (AUST). That's why we report the user balance for this
+                # strategy as usd_value / AUST price.
+                aust_asset = symbol_to_asset_or_token('AUST')
```

Comment:
if you know it's a particular token and it's in the DB then point to it as a constant from constants/asset.py.

Reply:
I'm not sure how to proceed here. I have AUST (Anchor UST) in my db so I think it also is in the rotki db, but it does not appear in `constants/asset.py`

Reply:
Yes, seems to be in the database. What you need to do is add it to `assets.py.template` and then use the script `tools/scripts/generate_constant_assets.py`. Then you can import AUST as `A_AUST` from the constants file

Reply:
Thx, unfortunately I'll get this error when running the script:

```
Traceback (most recent call last):
  File "rotki/tools/scripts/generate_constant_assets.py", line 145, in <module>
    main()
  File "rotki/tools/scripts/generate_constant_assets.py", line 124, in main
    generated_text += ctx.add_asset_initialization(var_name, identifier)
  File "rotki/tools/scripts/generate_constant_assets.py", line 21, in add_asset_initialization
    if asset_data.forked:
AttributeError: 'NoneType' object has no attribute 'forked'
```

Reply:
Did you use `A_AUST = Asset('AUST')`? By the error I would guess you used `EthereumToken`

Reply:
No, I used exactly that (Asset)

Reply:
Could it be that you deleted some asset from your database? Can you try using the packaged database and see if the error persists (take a backup and copy the packaged database where the global.db is)

Reply:
Sorry my time is running out today :) I'm not even sure which data directory it accesses. Why doesn't the script use the packaged database in the first place?

Reply:
rotki uses the database of the user if there is one, otherwise it creates a new one which is essentially a copy of the packaged database